reference/forms/types/options/datetime_choice_translation_domain.rst.inc

```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
```

Comment:
Unfortunately no. But maybe we can update the description of the existing option with what you used here (imo it's more clear than what we already have).